chainer_chemistry/dataset/preprocessors/common.py

```diff
@@ -165,3 +168,83 @@ def construct_discrete_edge_matrix(mol, out_size=-1):
         adjs[ch, i, j] = 1.0
         adjs[ch, j, i] = 1.0
     return adjs
+
+def construct_supernode_feature(mol, atom_array, adjs, out_size=-1):
+    """
+    Construct an input feature x' for a supernode
+
+    Args:
+        mol (rdkit.Chem.Mol): Input molecule
+        atom_array (numpy.ndarray) : array of atoms
+        adjs (numpy.ndarray): N by N 2-way array, or |E| by N by N 3-way array where |E| is the number of edgetypes.
+        out_size (int): not used...
+
+    Returns:
+        super_node_x (numpy.ndarray); 1-way array, the supernode feature.
+        len(super_node_x) will be 2 + 2 + MAX_ATOMIC_NUM*2 for 2-way adjs, 2 + 4*2 + MAX_ATOMIC_NUM*2 for 3-way adjs
+
+    """
+
+    largest_atomic_number = MAX_ATOMIC_NUM
```

Comment:
better to change it as argument? (we can change later)

Reply:
moved to a part of argument, with the default value